circuits/transfer/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "transfer-circuits"
-version = "0.2.0"
+version = "0.3.0"
 edition = "2021"
 
 [dependencies]
 rand_core = "0.6"
 
 dusk-bytes = "0.1"
-dusk-bls12_381 = "0.8"
-dusk-jubjub = "0.10"
-dusk-plonk = "0.9"
-dusk-poseidon = { version = "0.23.0-rc", features = ["canon"] }
-dusk-pki = "0.9.0-rc"
-dusk-schnorr = "0.9.0-rc"
-phoenix-core = { version = "0.15.0-rc", features = ["canon"] }
-canonical = "0.6"
+dusk-bls12_381 = "0.9"
+dusk-jubjub = "0.11"
+dusk-plonk = "0.10"
+dusk-poseidon = { version = "0.25.0-rc", features = ["canon"] }
+dusk-pki ="0.10.0-rc"
+dusk-schnorr = "0.10.0-rc"
+phoenix-core = { version = "0.16.0-rc", features = ["canon"] }
+canonical = "0.7"
 canonical_derive = { version = "0.6", optional = true }
```

Comment:
Thanks for catching this, `canonical_derive` didn't change at all between `0.6` and `0.7` but since it's part of the same crate we're bumping for consistency together; so yest it should be version "0.7" too.